alfresco-health-processor-platform/build.gradle

```diff
@@ -1,56 +1,27 @@
 plugins {
-    id "com.github.johnrengelman.shadow" version "7.1.0"
     id 'java-test-fixtures'
     id 'java-library'
     id 'io.freefair.lombok' version '6.2.0'
     id 'eu.xenit.alfresco'
     id 'eu.xenit.amp'
 }
 
-description = "Alfresco Repository module for ${rootProject.description}"
-
-apply from: "${rootDir}/gradle/internalPlatform.gradle"
-
-configurations {
-    ampLib {
-        extendsFrom(runtimeClasspath)
-    }
-    embedded {
-        canBeConsumed = false
-        canBeResolved = true
-        compileClasspath.extendsFrom(it)
-    }
-    runtimeElements {
-        outgoing.artifact(project.tasks.named('shadowJar'))
-    }
-    apiElements {
-        outgoing.artifact(project.tasks.named('shadowJar'))
-    }
-    alfrescoProvided {
-        extendsFrom(internalPlatform)
-    }
-}
+//description = "Alfresco Repository module for ${rootProject.description}"
```

Comment:
I don't understand why, but gradle couldn't add module.properties when the descriptions were set. Some bug in Alfresco Gradle SDK?

Reply:
```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':alfresco-health-processor-platform:testAmp'.

Caused by: org.gradle.api.GradleException: Could not add file '/git/xenit/alfresco-health-processor/alfresco-health-processor-platform/build/processTestModuleProperties/module.properties' to ZIP '/git/xenit/alfresco-health-processor/alfresco-health-processor-platform/build/dist/alfresco-health-processor-platform-0.5.7-SNAPSHOT-test.amp'.

Caused by: groovy.lang.GroovyRuntimeException: Failed to parse template script (your template may contain an error or be trying to use expressions not currently supported): startup failed:
SimpleTemplateScript4.groovy: 1: Unexpected character: '"' @ line 1, column 13.
   out.print("""module.description=Alfresco Repository module for Alfresco Health Processor\: A background processor capable of performing various operations on the nodes in your Alfresco repository, in a batched and controlled manner
```